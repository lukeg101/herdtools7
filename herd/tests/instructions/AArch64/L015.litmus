AArch64 L015

(* Tests STR barrel shifters - SXTW *)

{ uint64_t x; 0:x0=1; 0:x2=x; int32_t 0:x3=0;}
P0                        ;
   STR W0, [X2, W3, SXTW];

exists (x=0x1)
