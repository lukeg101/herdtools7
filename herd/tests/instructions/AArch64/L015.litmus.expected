Test L015 Allowed
States 1
x=1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition exists (x=1)
Observation L015 Always 1 0
Hash=1e61a83b6f7566c8526ca4540c009b2f

